test/payments_request_test.go

```diff
@@ -170,6 +120,30 @@ func TestMakeCardVerification(t *testing.T) {
 
 }
 
+func TestRequestPaymentList(t *testing.T) {
```

Comment:
In the future we should avoid using the old way of testing, and refactor to our current pattern